packages/twenty-server/src/engine/twenty-orm/workspace-schema-manager/services/workspace-schema-column-manager.service.ts

```diff
@@ -92,19 +92,38 @@ export class WorkspaceSchemaColumnManagerService {
     const safeSchemaName = removeSqlDDLInjection(schemaName);
```

Comment:
let's add Postgres type here and use it to take our decision. We should assume that defaultValues are correctly formed at this point (avoid typeof checks if it does not makes sense from business perspective)